### PR TITLE
Supporting null safety

### PR DESCRIPTION
**Pull request by KarimAbdo**

### CODE REVIEW

**Comment by abhishekUpmanyu** [comment]
Hi @KarimAbdo, thanks a lot for contributing to my repo.
Can you please update the version from 1.0.0 to 1.1.0 in pubspec.yaml
Also please remove those null checking assert statements for child and body.
Everything else looks perfect! :)

**Comment by BK1031** [comment]
So when is this getting merged?

File: lib/swipe_up.dart [hunk 1]
```diff
@@ -79,7 +79,7 @@ class _SwipeUpState extends State<SwipeUp> with SingleTickerProviderStateMixin {
       },
       onVerticalDragEnd: (details) {
         if (_swipeOffset > MediaQuery.of(context).size.height / 8) {
-          widget.onSwipe();
+          widget.onSwipe!();
```
Comment (bdlukaa) [inline lib/swipe_up.dart]:
```suggestion
          widget.onSwipe?.call();
```

**Review by bdlukaa — CHANGES_REQUESTED** [review]

**Comment by abhishekUpmanyu** [comment]
Sorry about the delay, merging the PR now

**Pull request merged**